bitnami/odoo/templates/pvc.yaml

```
@@ -8,8 +8,12 @@ metadata:
     {{- if .Values.commonLabels }}
     {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
     {{- end }}
+  annotations:
+  {{- if .Values.persistence.resourcePolicy }}
+  helm.sh/resource-policy: {{ .Values.persistence.resourcePolicy | quote }}
+  {{- end }}
```

Please indent all the `annotations` content. In the same way it is in the `labels` section
```suggestion
    {{- if .Values.persistence.resourcePolicy }}
    helm.sh/resource-policy: {{ .Values.persistence.resourcePolicy | quote }}
    {{- end }}
```